VERSIONS

```diff
@@ -1,3 +1,3 @@
-v1.2.2
-pkg/v2.4.3
-mongo/v2.2.2
+v1.2.3
+pkg/v2.4.4
```

Comment:
this should not be bumped as there is no change in the pkg module.

As of this commit, mongo version should also not be changed but I'm going to push a commit that bumps dependencies in mongo package, then this would be needed.